sema/commons/web/download_to_file.py

```
@@ -22,7 +22,7 @@ def download_to_file(
         dump_path=Path(dump_path),
         filename=filename,
         url=url,
-        mime_type=cgi.parse(response.headers.get("Content-Type"))[0],
+        mime_type=cgi.parse(response.headers.get("Content-Type"))[0],  # type: ignore # noqa
```

Ruff

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)